### ISSUE TITLE
bond module error check

### ISSUE THREAD

**Issue by jjhesk** [ixofoundation/ixo-blockchain#161]
According to the doc that is mentioned 
This message is expected to fail if:
```
...
reserve tokens list is faulty:
For swapper_function: two valid comma-separated denominations, e.g. res,rez
Otherwise: one or more valid comma-separated denominations, e.g. res,rez,rex
```
but the program was still able to create bonds by using `swapper_function`
```
--function-type=swapper_function \
--function-parameters="" \
--reserve-tokens=atoken,btoken \
```

successfully.

**Comment by migueldingli1997** [comment]
Ah, that's misleading! What the list is meant to show is the valid possibilities, rather than the invalid possibilities. Thanks @jjhesk, I'll re-word it.

**Comment by migueldingli1997** [comment]
Fixed in f66d046

**Comment by jjhesk** [comment]
@migueldingli1997 another error i got from the reserve token is that it must be the same types of tokens in full and not either one. noticed from https://github.com/ixofoundation/ixo-blockchain/blob/371e0abc26800486fee09d09e811546902431148/x/bonds/internal/types/bonds.go#L478

referencing by 
if the reserved tokens are declared atoken,btoken
`cli tx bonds buy [bond-token-with-amount] [max-prices] [bond-did] [buyer-did] [flags]`
the max prices must be 100atoken,100btoken.. 
if the user do not have btoken at all then it force him to state in the cmd???
